Mute Me Now/AppDelegate.m

```diff
@@ -28,16 +27,6 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
 
     [NSTouchBarItem addSystemTrayItem:mute];
     DFRElementSetControlStripPresenceForIdentifier(muteIdentifier, YES);
-
-    [self enableLoginAutostart];
```

Comment:
Automatically enabling "launch at login" is very user-hostile and I wouldn't recommend doing it. Let the user decide for themselves.

Also, Apple won't allow this in Mac App Store if you ever going to publish there.